src/ui-config/networksConfig.ts

```diff
@@ -226,6 +226,27 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
       url: 'https://bridge.arbitrum.io',
     },
   },
+  [ChainId.arbitrum_sepolia]: {
+    name: 'Arbitrum Sepolia',
+    publicJsonRPCUrl: [
+      'https://sepolia-rollup.arbitrum.io/rpc',
+      'https://public.stackup.sh/api/v1/node/arbitrum-sepolia',
+    ],
+    publicJsonRPCWSUrl: 'https://sepolia-rollup.arbitrum.io/rpc',
+    baseUniswapAdapter: '0x0',
+    baseAssetSymbol: 'ETH',
+    wrappedBaseAssetSymbol: 'WETH',
+    baseAssetDecimals: 18,
+    explorerLink: 'https://sepolia.arbiscan.io',
+    // usdMarket: true,
```

Comment:
Could probably just delete all of the commented out config, it's not relevant for testnets